tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md

```diff
@@ -21,6 +21,9 @@
       - [Certificate Authority Bundles (28)](#Certificate-Authority-Bundles-28)
     - [Operator Framework / operator-lifecycle-manager (2)](#Operator-Framework-/-operator-lifecycle-manager-2)
       - [Certificates (2)](#Certificates-2)
+    - [Unknown (9)](#Unknown-9)
+      - [Certificates (3)](#Certificates-3)
+      - [Certificate Authority Bundles (6)](#Certificate-Authority-Bundles-6)
```

Comment:
New generated markdown for file-based certs and CA bundles here.

Reply:
Updated